docs/en/operation/Journal.md

```diff
@@ -93,3 +94,49 @@ If the restore succeeds, you should see a log message along the lines of
 INFO AlluxioMasterProcess - Restored 57 entries from backup
 ```
 in the primary master logs.
+
+### Changing Masters
+
+If the journal is stored in a distributed system like HDFS, changing masters is easy.
```

Comment:
I guess the emphasis is shared storage rather than distributed system?

Reply:
updated